test.sh

```diff
@@ -16,7 +16,10 @@
 
 source base.sh
 
-#Test for codeworld-compiler
-run codeworld-compiler cabal configure --enable-tests
-run codeworld-compiler cabal build 
-run codeworld-compiler cabal test
+DirList="codeworld-compiler"
```

Comment:
Oops, my mistake!  I meant `dirname`.  Your version works, but `dirname` is probably clearer.